lighthouse-core/config/default.json

```diff
@@ -285,6 +287,13 @@
         }
       }
     }, {
+      "name": "Using bytes efficiently",
+      "audits": {
+        "unused-css-rules": {
+          "expectedValue": false
```

Comment:
followup: all the expectedValue entries here seem to be useless for non-scored sections, the report just wants `true` for success and `false` for failure, I can toggle this all day with no change in the report, but the `rawValue === didPass`